corehq/apps/user_importer/helpers.py

```diff
@@ -142,10 +142,12 @@ def __init__(self, upload_domain, user_domain, user, upload_user, is_new_user, v
 
         self.role_updated = False
 
-    def update_role(self, role_qualified_id):
+    def update_role(self, role_qualified_id, clear_role=False):
         user_current_role = self.user.get_role(domain=self.user_domain)
         self.role_updated = not (user_current_role
                                  and user_current_role.get_qualified_id() == role_qualified_id)
+        if user_current_role and clear_role:
```

Comment:
I don't think you need to pass in `clear_role`.  When you pass in `'none'` as the `role_qualified_id` then `self.role_updated` is going to get set to `True` in the line above.

Reply:
You're right, removing `clear_role`.